File: questions/maxInArray/ans.py
result = []
for i in range(0,int(input())):
    n = int(input())
    maxh = -float('inf')
    for _ in range(n):
        h = int(input())
        if h > maxh:
            maxh = h
    result.append(maxh)
print("output:")
for i in result:
    print(i)

File: questions/maxInArray/ans.cpp
#include <iostream>
using namespace std;

int main()
{
	int t, n, h, maxh[10];
	cin >> t;
	for (int i = 0; i < t; i++)
	{
		cin >> n;
		maxh[i] = 0;
		for (int j = 0; j < n; j++)
		{
			cin >> h;
			if (h > maxh[i])
				maxh[i] = h;
		}
	}

	for (int a = 0; a < t; a++)
		cout << maxh[a] << endl;

	return 0;
}
